src/main/resources/fabric.mod.json

```diff
@@ -31,12 +31,13 @@
   ],
 
   "depends": {
-    "fabricloader": ">=0.11.3",
+    "fabricloader": ">=0.15.0",
     "fabric": ">=0.84.0",
     "minecraft": ">=1.20.1",
     "java": ">=17"
   },
   "breaks": {
-    "sodium": "<0.5.0"
+    "sodium": ["<0.5.0", "<0.6 >=0.6.0-beta.2"],
```

Comment:
Is this correct? Based on the specification, this means "breaks if Sodium is (<0.5.0 OR (<0.6 AND >=0.6.0-beta.2))", which doesn't make sense because Sodium cannot be older than 0.6 and newer than or equal to 0.6.0-beta.2 at the same time. Also, use `0.6.0` instead of `0.6`.

Reply:
Correction: Sodium can be <0.6.0 and newer than or equal to 0.6.0-beta.2 at the same time because 0.6.0-beta.2 is a prerelease version; however, this still doesn't make sense because this range would break with all versions newer than beta 2 instead of older than beta 2 as it is supposed to.

Reply:
`>0.6.0 <0.6.0-beta.2` should work then i think? if greater than 0.6.0 and less than 0.6.0-beta.2?

Reply:
No, because a version cannot be greater than 0.6.0 and less than 0.6.0-beta.2 at the same time.

Reply:
I think `["<0.5.0", "~0.6.0 <0.6.0-beta.2"]` will work correctly. Please check that this is actually the case though; try testing this predicate with `0.4.9`, `0.5.11`, `0.6.0-beta.1`, `0.6.0-beta.2`, and `0.6.0` specifically in Minecraft as online tools may not match Fabric Loader's behavior.

Reply:
`VersionPredicate.parse("~0.6.0 <0.6.0-beta.2").test(Version.parse("0.6.0-beta.1"))` returns false when it should be returning true so something else needs to be done for the 0.6.0 breaks